.github/workflows/ci.yml

```diff
@@ -17,27 +17,20 @@ jobs:
     strategy:
       matrix:
         include:
-          - mediawiki_version: '1.35'
-            smw_version: '4.1.2'
-            php_version: 7.4
-            database_type: mysql
-            database_image: "mysql:5.7"
-            coverage: true
-            experimental: false
-          - mediawiki_version: '1.35'
+          - mediawiki_version: '1.39'
             smw_version: dev-master
-            php_version: 7.4
+            php_version: 8.1
             database_type: mysql
-            database_image: "mysql:5.7"
+            database_image: "mariadb:11.2"
             coverage: false
             experimental: false
           - mediawiki_version: '1.39'
             smw_version: dev-master
             php_version: 8.1
             database_type: mysql
-            database_image: "mariadb:latest"
-            coverage: false
-            experimental: true
+            database_image: "mariadb:11.2"
+            coverage: true

```

Comment:
Coverage won't work with MW 1.39. It will work with MW 1.40.

Reply:
Thanks, done.